tests/letstest/targets.yaml

```diff
@@ -1,6 +1,21 @@
 targets:
   #-----------------------------------------------------------------------------
   #Ubuntu
+  - ami: ami-064bd2d44a1d6c097
```

Comment:
Why did you choose the us-east targets? Why not west? Do the test farm tests always run on east? Is that something users set up manually?

Reply:
Yes, the test farm tests currently always run on US east. Which region you use is something you configure when you set up your AWS credentials with `aws configure`. Our instructions in the release notes for setting up these tests have you choose us-east-1 because that's where all AMIs in this file are located.